<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HBSRecover</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../hbsrecover.qrc">
    <normaloff>:/resources/hbsrecovery.png</normaloff>:/resources/hbsrecovery.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="srcpath_label">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="srcpath"/>
        </item>
        <item>
         <widget class="QToolButton" name="srcpath_btn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="subfolders_cb">
          <property name="text">
           <string>Include subfolders</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dstpath_label">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="dstpath"/>
        </item>
        <item>
         <widget class="QToolButton" name="dstpath_btn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pswd_label">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="pswd">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pswd_cb">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="filelist">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="recover">
        <property name="text">
         <string>Recover</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_sel_src"/>
    <addaction name="action_sel_dst"/>
    <addaction name="action_recover"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about_hbsrecover"/>
    <addaction name="action_about_qt"/>
    <addaction name="separator"/>
    <addaction name="action_report_an_issue"/>
    <addaction name="action_star_me_on_github"/>
    <addaction name="action_license"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_sel_dst">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/code-greater-than.svg</normaloff>:/resources/material-design/svg/code-greater-than.svg</iconset>
   </property>
   <property name="text">
    <string>Select Destination...</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/exit-to-app.svg</normaloff>:/resources/material-design/svg/exit-to-app.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_recover">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/folder-lock-open.svg</normaloff>:/resources/material-design/svg/folder-lock-open.svg</iconset>
   </property>
   <property name="text">
    <string>Recover</string>
   </property>
  </action>
  <action name="action_about_hbsrecover">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/hbsrecovery.png</normaloff>:/resources/hbsrecovery.png</iconset>
   </property>
   <property name="text">
    <string>About HBSRecover...</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/qt_extended_48x48.png</normaloff>:/resources/qt_extended_48x48.png</iconset>
   </property>
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="action_star_me_on_github">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/github-box.svg</normaloff>:/resources/material-design/svg/github-box.svg</iconset>
   </property>
   <property name="text">
    <string>Star me on Github...</string>
   </property>
  </action>
  <action name="action_sel_src">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/code-less-than.svg</normaloff>:/resources/material-design/svg/code-less-than.svg</iconset>
   </property>
   <property name="text">
    <string>Select Source...</string>
   </property>
  </action>
  <action name="action_license">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/book-open.svg</normaloff>:/resources/material-design/svg/book-open.svg</iconset>
   </property>
   <property name="text">
    <string>License information...</string>
   </property>
  </action>
  <action name="action_report_an_issue">
   <property name="icon">
    <iconset resource="../hbsrecover.qrc">
     <normaloff>:/resources/material-design/svg/bug.svg</normaloff>:/resources/material-design/svg/bug.svg</iconset>
   </property>
   <property name="text">
    <string>Report an issue...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../hbsrecover.qrc"/>
 </resources>
 <connections/>
</ui>
